README.md

```diff
@@ -34,6 +34,17 @@ Bazelisk currently understands the following formats for version labels:
 
 In the future we will add support for building Bazel from source at a given commit.
 
+## How does Bazelisk download release from a fork?
+
+It uses a simple algorithm:
+- If the environment variable `USE_BAZEL_REMOTE` is set, it will use the fork name specified in the value.
```

Comment:
Can we call this `USE_BAZEL_FORK` (and respectively below replace "remote" with "fork" in other places, too)?

While I understand where the name "remote" comes from, it's already a pretty often used term in Bazel (e.g. for "remote execution" and "remote caching"). I worry a bit that there might be confusion around this.

Reply:
It's a good idea. Thanks.